specs/global/interfaces.json

```diff
@@ -0,0 +1,129 @@
+{
+    "Fingerprintable": {
+        "type": "object",
+        "properties": {
+            "labels": {
+                "$ref": "./schemas.json#/Labels"
```

Comment:
This feels a bit like a violation to me, but I can't explain why...

Reply:
Please elaborate. Something specifically about `labels` or the whole interface idea?

Reply:
The fact that this particular interface relies on implementations (that is, `schema.json`). Then again It might really just be in my head..

Reply:
I think I know what you mean. In an interface the right hand sides (values) should probably also be described by interfaces. It's doable - `labels` is a (language-based) map of `Term`s and would not conflict with `schema.json`'s definition of `labels`.

But that's maybe where the interface analogy ends. `schema.json`'s `Entity` is not using `Fingerprintable` as interface but as mixin/trait which, if we want to have concrete `Labels`, ... types, should hold these.

Maybe have yet another file for those behavior/reuse types?

Reply:
Well, Entity is slated to move to the interfaces file as soon as we separate items from properties. To me (in this particular case) an interface is pretty much a mixin. I would vote of ambiguating the current file name over adding another file (which might raise the complexity level... maybe we could call this `mixins.json` instead of interfaces which is, granted, a pretty loaded term. What do you say?

Reply:
> Well, Entity is slated to move to the interfaces file as soon as we separate items from properties.

If that is a given then let's do this first and have the discussion afterwards. Maybe things are more obvious then.

Reply:
:+1: